### PR TITLE
Fix small grammatical error in on-facebook-p2 message

### PR DESCRIPTION
**Pull request by euphemism**

### CODE REVIEW

**Review by flodolo — COMMENTED** [review]
Ugh, that's definitely a typo. I'm OK with fixing also the other en-* variants, but flagging one of the devs for visibility.

I wonder if "need access to" is better than "need to access".

**Comment by euphemism** [comment]
> Ugh, that's definitely a typo. I'm OK with fixing also the other en-* variants, but flagging one of the devs for visibility.
> 
> I wonder if "need access to" is better than "need to access".

I personally would have probably gone with "need access to," but I was bringing the message inline with the content of `onboarding2-p2`.

**Review by peeteyg — APPROVED** [review]

**Comment by flodolo** [comment]
@peeteyg 

This is not a playground, don't go around messing with other people's work. Thanks.

**Comment by maxxcrawford** [comment]
@flodolo Great call-out. By editing the existing string (and not making a new one) does that stop it from having to be localized else where?

File: src/_locales/en-CA/messages.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "description": "This is shown in the body of the panel pop-up when the user is on a Facebook website, which has been automatically opened into Facebook Container. DO NOT TRANSLATE \"Facebook Container\"."
   },
   "on-facebook-p2": {
-    "message": "These sites need to access to your Facebook identity (or know who you are) in order to work. This also means that Facebook can track everything you do on these sites.",
+    "message": "These sites need to access your Facebook identity (or know who you are) in order to work. This also means that Facebook can track everything you do on these sites.",
```
Comment (maxxcrawford) [inline src/_locales/en-CA/messages.json:28]: See the following suggestion. Please make these across all three strings. 

```suggestion
    "message": "These sites need access to your Facebook identity (or know who you are) in order to work. This also means that Facebook can track everything you do on these sites.",
```

File: src/_locales/en-GB/messages.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "description": "This is shown in the body of the panel pop-up when the user is on a Facebook website, which has been automatically opened into Facebook Container. DO NOT TRANSLATE \"Facebook Container\"."
   },
   "on-facebook-p2": {
-    "message": "These sites need to access to your Facebook identity (or know who you are) in order to work. This also means that Facebook can track everything you do on these sites.",
+    "message": "These sites need to access your Facebook identity (or know who you are) in order to work. This also means that Facebook can track everything you do on these sites.",
```
Comment (maxxcrawford) [inline src/_locales/en-GB/messages.json:28]:
```suggestion
    "message": "These sites need access to your Facebook identity (or know who you are) in order to work. This also means that Facebook can track everything you do on these sites.",
```

File: src/_locales/en/messages.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
         "description": "This is shown in the body of the panel pop-up when the user is on a Facebook website, which has been automatically opened into Facebook Container. DO NOT TRANSLATE \"Facebook Container\"."
     },
     "on-facebook-p2": {
-      "message": "These sites need to access to your Facebook identity (or know who you are) in order to work. This also means that Facebook can track everything you do on these sites.",
+      "message": "These sites need to access your Facebook identity (or know who you are) in order to work. This also means that Facebook can track everything you do on these sites.",
```
Comment (maxxcrawford) [inline src/_locales/en/messages.json:28]:
```suggestion
      "message": "These sites need access to your Facebook identity (or know who you are) in order to work. This also means that Facebook can track everything you do on these sites.",
```

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
@euphemism Thanks for catching this error. 

@flodolo Thanks for breaking this to our attention!

**Comment by flodolo** [comment]
> @flodolo Great call-out. By editing the existing string (and not making a new one) does that stop it from having to be localized else where?

No, what counts is the string ID (`on-facebook-p2`), so we can merge this without impacting other existing localizations.

**Pull request merged**